tools/lkl/include/lkl.h

```diff
@@ -229,6 +233,17 @@ struct lkl_netdev {
 int lkl_netdev_add(struct lkl_netdev *nd, void *mac);
 
 /**
+* lkl_netdevs_remove - destroy all network devices
```

Comment:
Hmmmm I think this was just not a good way to organize this. I did it this way because `lkl.h` isn't available from `arch/lkl` but between your comments and Xiao's I think it's probably better to just move those function prototypes into somewhere that's visible from `arch/lkl`.